src/main/webapp/app/entities/cancer-type/cancer-type-update.tsx

```diff
@@ -45,12 +41,13 @@ export const CancerTypeUpdate = (props: ICancerTypeUpdateProps) => {
     }
   }, [updateSuccess]);
 
+  // TYPE-ISSUE: I don't know what type values is
```

Comment:
It technically should be the type of the entity (ie `ICancerType`), but the values come from the `ValidatedField`, which can be any type?

Reply:
Same goes for all the `saveEntity` methods. The type should be the entity model type.

Reply:
for line 50 there is a call to `parentId` like so `values.parentId.toString()` which doesn't exist on the `ICancerType` interface.